code/modules/mob/mob.dm

```diff
@@ -106,6 +106,11 @@
 // blind_message (optional) is what blind people will hear e.g. "You hear something!"
 
 /mob/visible_message(message, self_message, blind_message)
+	if(!isturf(src.loc)) // cant see what do mob do in a locker or something
```

Comment:
```suggestion
	if(!isturf(loc)) //mobs inside objects (such as lockers) shouldn't have their actions visible to those outside the object
```
src is implied here.  Also your comment needs to be more clear

Reply:
removed in update
comment changed in update